### PR TITLE
Fixed a few typos and errors

### PR DESCRIPTION
**Pull request by stevencl**
I copied and pasted each of the code snippets and found a few typos and other errors which I have attempted to fix.

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Keys/README.md [hunk 1]
```diff
@@ -182,7 +185,7 @@ Console.WriteLine(key.DeletedDate);
 This example lists all the keys in the specified Key Vault.
 
 ```c#
-IEnumerable<Response<KeyBase>> allKeys = client.GetKeys();
+IEnumerable<Azure.Response<KeyBase>> allKeys = client.GetKeys();
```
Comment (schaabs) [inline sdk/keyvault/Azure.Security.KeyVault.Keys/README.md]: I'm not a huge fan of fully qualified namespaces in the sample code. I wonder if there's some way to document the using statements needed to not fully qualify

**Review by schaabs — APPROVED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Keys/README.md (hunk 1 above)
Comment (stevencl) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/README.md]: Agreed, my mistake. We can just put

```c#
using Azure;
```

above the snippet I think?

**Review by AlexGhiondea — APPROVED** [review]

**Comment by pakrym** [comment]
Is this still relevant?

**Comment by heaths** [comment]
No. There is no more `KeyBase` and the README has been updated for GA.

@stevencl, thank you for fixing these issues prior! I've made sure our samples build and run nightly, and our READMEs are populated directly from those samples along with verification during PR/CI builds they match samples. Going forward, this shouldn't happen again. Thank you!

**Pull request closed without merging**